go.mod

```diff
@@ -8,14 +8,15 @@ require (
 	github.com/kubewarden/kubewarden-controller v1.9.0
 	github.com/rs/zerolog v1.31.0
 	github.com/spf13/cobra v1.8.0
+	golang.org/x/exp v0.0.0-20231206192017-f3f8817b8deb
 	k8s.io/api v0.29.0
 	k8s.io/apimachinery v0.29.0
 	k8s.io/client-go v0.29.0
 	sigs.k8s.io/controller-runtime v0.16.3
-	sigs.k8s.io/wg-policy-prototypes v0.0.0-20230505033312-51c21979086a
```

Comment:
This represents a bump in the wg-policy-prototypes that changes the field of the crds. See the comparison here:
https://github.com/kubernetes-sigs/wg-policy-prototypes/compare/51c21979086a654f60d5d65a95f9681e954816ac..22764d64b0c3f79d2293b67000b4d9ebca197623

Particularly the file `policy-report/crd/v1beta1/wgpolicyk8s.io_clusterpolicyreports.yaml`. There's only 2 new fields to the CRDs, and no deletions, but that means that PolicyReports created with this new library version may not be backwards compatible.

Thing is, https://github.com/kubernetes-sigs/wg-policy-prototypes is not correctly tagged nor has proper releases, and other community consumers keep just developing the future v1beta1 (listed as [WIP](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report#policy-report)), so we need to be careful.

I would like to postpone this bump.

Reply:
oh, I see! that makes sense, let me revert that change in a bit